test/spec/fixtures/constants.js

```diff
@@ -0,0 +1,12 @@
+window.OPENSHIFT_CONSTANTS = window.OPENSHIFT_CONSTANTS || {};
+
+window.OPENSHIFT_CONSTANTS.AVAILABLE_KINDS_BLACKLIST = [
+    // These are k8s kinds that are not supported in the current release of OpenShift
+    { kind: "Binding", group: "servicecatalog.k8s.io" },
```

Comment:
Update the comment so its clear that its only being blacklisted for testing purposes, the comment as-is is definitely misleading, we won't be blacklisting the `servicecatalog.k8s.io` Binding in the console.